latex/bapc.cls

```diff
@@ -167,17 +169,6 @@
 \newenvironment{Explanation}{
 	\subsection*{\langexplanation{}}
 }{\par}
```

Comment:
And while we're at it, what was the `Explanation` environment used for again?

Reply:
not sure about that, never used it

Reply:
let's drop if you didn't already.

Reply:
but we even added this to the lang files... i would prefer to keep it actually.
And even with the new abilities it might be useful?